.github/workflows/build_linux_pytorch_wheels.yml

```diff
@@ -56,13 +58,13 @@ jobs:
           echo "Building PyTorch wheels for ${{ inputs.target }}"
           ./external-builds/pytorch/build_prod_wheels.py \
             --pip-cache-dir /tmp/pipcache \
+            --find-links "${{ inputs.find_links }}" \
             build \
-              --find-links "https://therock-nightly-python.s3.us-east-2.amazonaws.com/${{ inputs.family }}/index.html" \
-              --install-rocm \
-              --clean \
-              --output-dir ${{ env.PACKAGE_DIST_DIR }} \
-              --pytorch-rocm-arch ${{ inputs.target }} \
-              --version-suffix "+rocmsdk${{ inputs.version_suffix }}"
+            --install-rocm \
+            --clean \
+            --output-dir ${{ env.PACKAGE_DIST_DIR }} \
+            --pytorch-rocm-arch ${{ inputs.target }} \
+            --version-suffix "+rocmsdk${{ steps.version.outputs.version_suffix }}"
```

Comment:
What purpose does the `set_version_suffix.py` script serve?

If you omit the `--version-suffix` argument here, `build_prod_wheels.py` will already have the behavior that the new script is adding:

```python
    today = date.today()
    formatted_date = today.strftime("%Y%m%d")
    build_p.add_argument(
        "--version-suffix",
        default=f"+rocmsdk{formatted_date}",
        help="PyTorch version suffix",
    )
```

It can be useful to compute a version in a workflow file instead of in a script if you want the version to change based on the context of the workflow, e.g. when selecting between "dev", "release candidate", and "stable", like here: https://github.com/nod-ai/shark-ai/blob/b95ab3b578c0016f4f86121fc9e41dbc5fabe5f5/.github/workflows/build_packages.yml#L49-L64

Reply:
https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it . If the script isn't doing anything right now, remove it.

Reply:
Can you please let me know which one you want? Remove  or keep it?

Reply:
Sound good!, since it has the same behavior without the flag - no need to add extra complexity.